docs/curating/variants.rst

```diff
@@ -1,4 +1,11 @@
 Curating Variants
 =================
 
-A CIViC Variant represents any molecular alteration with evidence for clinical relevance in cancer. A new variant is added to the CIViC database when the first Evidence Item for that variant is submitted. Once a variant exists, with one or more evidence items, variant coordinates and other variant-level data can also be curated. The CIViC definition of a variant is intentionally broad to encompass not only simple variation (e.g., SNVs and indels), but also regional/categorical variation (e.g., exon mutation), or other types of variation (e.g., expression, fusions, etc.). Features within the Variant knowledge model include: Variant Summary, Alias, HGVS expressions, ClinVar IDs, Sources, Variant Type, representative Variant Coordinates and Transcript. Variants are linked to associated CIViC Assertions, Variant Evidence Score, and external data from MyVariant.info. Additional instruction on curating individual components of variants are provided in the CIViC Knowledge Model :ref:`variants` docs.
+A CIViC Variant represents any molecular alteration with evidence for clinical relevance in cancer. A new variant is added to the CIViC database *when the first Evidence Item for that variant is submitted*.
+
+Once a variant exists, with one or more evidence items, variant coordinates and other variant-level data can also be curated. The CIViC definition of a variant is intentionally broad to encompass not only simple variation (e.g., SNVs and indels), but also regional/categorical variation (e.g., exon mutation), or other types of variation (e.g., expression, fusions, etc.).
```

Comment:
```suggestion
Once a variant exists, with one or more evidence items, variant coordinates and other variant-level data can also be curated. The CIViC definition of a variant is intentionally broad to encompass not only simple variation (e.g., SNVs and indels), but also regional/categorical variation (e.g., exon mutation), or other types of variation (e.g., expression, fusions).
```

`etc.` is redundant in a list of examples, since `e.g.` implies that the list is non-exhaustive. (Otherwise, `i.e.` would be appropriate instead.)